src/index.ts

```diff
@@ -23,12 +24,8 @@ class MyClient extends AkairoClient {
 export const client = new MyClient();
 
 const options: ConnectionOptions = {
+  url: DATABASE_URL,
   type: 'postgres',
-  host: DB_HOST,
-  port: DB_PORT,
-  username: DB_USERNAME,
-  password: DB_PASSWORD,
-  database: DB_NAME,
   entities: [`${__dirname}/entity/*.js`],
   synchronize: true,
   logging: false,
```

Comment:
`logging` should be `true` in development mode, so we can see the SQL queries in logs. Really helpful to check number of queries each request triggers in backend and what are the exact queries. It can be turned OFF in production mode.

We can set it to `logging: process.env.NODE_ENV !== 'production'`.